packages/devextreme/js/exporter/image_creator.js

```diff
@@ -455,7 +455,7 @@ function applyGradient(context, options, { linearGradients, radialGradients }, e
         });
 
         if(type === 'linear') {
-            const angle = gradients[id].transform?.replace(/\D/g, '') * Math.PI / 180 ?? 0;
+            const angle = (gradients[id].transform?.replace(/\D/g, '') || 0) * Math.PI / 180;
```

Comment:
suggestion: let's add convertion string to number to multiply numbers only

Reply:
sounds reasonable but let's avoid reruning all the tests for such a minor task